### DISCUSSION TITLE
control server authentication via env vars

### DISCUSSION THREAD

**Discussion by usma0118** [qdm12/gluetun#2571] [Q&A] [answered]
I am looking into configuring https://github.com/qdm12/gluetun-wiki/blob/main/setup/advanced/control-server.md#authentication

Can this be configured via environment vars?

**Comment by usma0118** [comment]
@qdm12 any info?

**Comment by qdm12** [comment, marked as answer]
Done in https://github.com/qdm12/gluetun/commit/3fac02a82ac26206b9c6d76e75e7ee1526ca4b87
